### ISSUE TITLE
mesh networking - init RF missing?

### ISSUE THREAD

**Issue by JanneKiiskila** [ARMmbed/easy-connect#9]
Should one init the RF PHY 1st before connecting?

```
#ifdef MESH
    output.printf("\n\rConnecting to Mesh...\r\n");
    mesh.initialize(&rf_phy);
    connect_success = mesh.connect();
    network_interface = &mesh;
#endif
```

**Comment by JanneKiiskila** [comment]
This was taken care with my mbed-client submit.

**Comment by janjongboom** [comment]
Landed in #11.
